templates/emails/activation_email.txt

```diff
@@ -0,0 +1,38 @@
+<%namespace file="../main.html" import="stanford_theme_enabled" />
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+
+% if is_secure:
+  <a href="https://${ site }/activate/${ key }">
+% else:
+  <a href="http://${ site }/activate/${ key }">
+% endif
+
+    <img src="http://courses.gymna.si/static/themes/gymnasium/images/gym-reg-activate.png" alt="click to activate">
```

Comment:
Actually - so lines 6-10 will work... the <img> tag very likely won't work, but since devstack only outputs the reg email to the console, I wasn't able to test it (beyond the console showing the above as plaintext).  Based on [this PR](https://github.com/edx/edx-platform/pull/6681), I don't think the img tag will work, unless you know better.  I can submit a change to revert line 12 in such a case.  The lines prior were already in the template from edx-platform's default files.